storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -2210,9 +2212,17 @@ decrypt_decompress_file(const char *filepath, uint thread_n)
  		message << "decompressing";
  		needs_action = true;
  	}
-
-	cmd << " > " << SQUOTE(dest_filepath);
- 	message << " " << filepath;
+	message << " " << filepath;
+        output_file << "xboutputfile" << thread_n;
+
+	if (setenv(output_file.str().c_str(), dest_filepath, 1) == -1) {
+		int errsv = errno;
+		msg_ts("[%02u] Can not set env %s: to %s got error %d\n",
+				thread_n, output_file.str().c_str(),
+				dest_filepath, errsv);
+		return false;
+	}
+	cmd << " >\"$" << output_file.str().c_str() << "\"";
```

Comment:
Please add a comment how the final command line will look like

Reply:
added